lib/jira/resource/agile.rb

```diff
@@ -48,6 +48,9 @@ def path_base(client)
         self.class.path_base(client)
       end
 
+      def self.hash_to_query_string(query_params)
```

Comment:
I just saw you comment in the PR

> One thing I was not quite sure on, was how to generate the options element of the URI. I found something in the Base class, but could not find a way to use it in JIRA::Resource::Agile class. Pls let me know, if there is a better way.

It is probably due to the fact that `hash_to_query_string` was an instance method that you wanted to use in a class method.
However, there is now the class method too 😉